srcpkgs/dmd/template

```diff
@@ -15,13 +15,13 @@ distfiles="
  https://github.com/dlang/druntime/archive/v${version}.tar.gz>druntime-${version}.tar.gz
  https://github.com/dlang/phobos/archive/v${version}.tar.gz>phobos-${version}.tar.gz
  http://downloads.dlang.org/releases/2.x/${version}/dmd.${version}.linux.tar.xz"
-checksum="bcca38f4b80b51ae0a1955dc29e6cbfaa4d01d94869ddfcacf9292898c34343a
- cc8ba196b08cca04488d97d0e35c878f7351b7d8128aec90eaa312d2bd9a6af2
- df9f81eecda4366adc5ca9ed31a0b4474c248544b120ff6fa3f6cb45917d8769
- 6e4bd4ee73a553921cdca7e3e21768b842b18186d07c0b4898fbd13a359875c0"
+checksum="63271e939dbe5c39e866074b97f53c515c2b9161fcae9bae1df3b38924b0098d
+ 4bd91e438996363ec62f0b72392fa88f9d53ee73c51bea1c3be1ae97bd28d30c
+ e84536c538f47247d4803a9d2163f32e4b1e03acac090fa195a77083147c556f
+ 5208ef1babdb17bf63324d3fd439af774e4bb90d242d55c2471775deba3a399b"
 conf_files="/etc/dmd.conf"
 provides="d-compiler-${version}_${revision}"
-conflicts="dmd-bootstrap dmd2.081"
```

Comment:
Can we replace any of these in the bootstrap chain?

Reply:
No. `dmd-bootstrap` is needed to build `2.081` (obviously), `2.081` is needed to build `2.091`  (and probably older versions, not sure what range is but it doesn't matter because...) `2.091` is needed to build anything past `2.092.1` (thanks to a  change in `mtype.d`).